source/Nevermore/Nevermore.csproj

```diff
@@ -29,6 +29,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Nevermore.Contracts\Nevermore.Contracts.csproj" />
+    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
```

Comment:
_I'm happy to remove this if you think the cost of taking on this dependency outweighs the benefits it may provide._

Reply:
Yeh I think so. If anything we'd use the new C# 8.0 feature